homestyle.css

```diff
@@ -0,0 +1,101 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap');
+
+* {
+	box-sizing: border-box;
+}
+
+body { 
+	margin: 0; 
+	padding: 0;
+	font-family: 'Montserrat', sans-serif;
+}
+
+.navbar {
+	width: auto;
+	max-width: 100%;
+	max-height: auto;
+	color: white;
+	background-color: black;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	padding: 10px;
+	text-decoration: none;
+	letter-spacing: 3px;
+	font-size: 25px;
+}
+
+#NC {
+	background-color: black;
+	color: white;
+	text-transform: uppercase;
+	padding: 15px;
+	
+}
+
+.links {
+	display: flex;
+	margin: 10px;
+	
+}
+
+.links a {
+  background-color: black;
+  border: #565656 solid 5px
+  color: white;
+  padding: 15px 50px;
+  text-align: center;
+  text-decoration: none;
+  font-size: 20px;
+  text-transform: uppercase;
+  letter-spacing: 3px;
+  font-family: 'Montserrat', sans-serif;
+}
+
+a:link{
+  color:white;
+  text-decoration: none;
+}
+a:visited{
+  color:white;
+  text-decoration: none;
+}
+a:hover{
+  color:white;
+  text-decoration: none;
+}
+a:focus{
+  color:white;
+  text-decoration: none;
+}
+a:active{
+  color:white;
+  text-decoration: none;
+}
+
+
+.homequote {
+    position: absolute;
```

Comment:
Might be good to make this quote move relative to screen size instead, as it stays in one place.